code/modules/mob/mob.dm

```diff
@@ -567,9 +583,21 @@
 	to_chat(src, examine_block(span_infoplain(result_combined)))
 	SEND_SIGNAL(src, COMSIG_MOB_EXAMINATE, examinify)
 
-/mob/proc/blind_examine_check(atom/examined_thing)
+//This is a proc for worn item examines. See /obj/item/proc/get_examine_line
+/mob/proc/run_worn_examinify_checks(atom/examinify)
```

Comment:
Change this to can_examine_in_detail or something similar and add a silent flag which sends no chat messages. The more generic a function is, the more applicable it is in other scenarios, every time you are adding a new proc is a scenario that isn't handled by existing procs and there may be scenarios in the future that need something that serves a similar purpose.